README.md

```diff
@@ -99,7 +99,9 @@ Below is a list of rules that are not enabled by default together with the reaso
 | [`avoid_annotating_with_dynamic`](https://dart.dev/tools/linter-rules/avoid_annotating_with_dynamic)                                                       | Not specified                                                                                                                |
 | [`avoid_catches_without_on_clauses`](https://dart.dev/tools/linter-rules/avoid_catches_without_on_clauses)                                                 | [Has unresolved false positives](https://github.com/dart-lang/linter/issues/3023)                                            |
 | [`avoid_classes_with_only_static_members`](https://dart.dev/tools/linter-rules/avoid_classes_with_only_static_members)                                     | Not specified                                                                                                                |
+| [`avoid_futureor_void`](https://dart.dev/tools/linter-rules/avoid_futureor_void)                                                                           | Not specified                                                                                                                |
 | [`avoid_implementing_value_types`](https://dart.dev/tools/linter-rules/avoid_implementing_value_types)                                                     | Not specified                                                                                                                |
+| [`avoid_null_checks_in_equality_operators`](https://dart.dev/tools/linter-rules/avoid_null_checks_in_equality_operators)                                   | Not specified                                                                                                                |
```

Comment:
Is there any exclusion reason for this removal we could add?

Reply:
It was deprecated in favor of `non_nullable_equals_parameter`